src/traces/parcoords/parcoords.js

```diff
@@ -557,7 +557,7 @@ module.exports = function(root, svg, styledData, layout, callbacks) {
                     .outerTickSize(2)
                     .ticks(wantedTickCount, d.tickFormat) // works for continuous scales only...
                     .tickValues(d.ordinal ? // and this works for ordinal scales
-                        sdom.map(function(d, i) {return texts && texts[i] || d;}) :
+                        sdom.map(function(d, i) {return texts && texts[i] !== undefined ? texts[i] : d;}) :
```

Comment:
> Is it OK or would you rather that a missing ticktext array or missing ticktext[i] yields an 'undefined' as the tick label?

Ideally, it would be nice to make parcoords reuse some of the `cartesian/axes.js` routines.

Now, should we consider the current cartesian behavior a bug? I'd vote yes. I'd think only `typeof strings` ticktext values should be visible cc @alexcjohnson